src/components/AlbumName/AlbumName.jsx

```diff
@@ -0,0 +1,18 @@
+import "./albumname.css";
+
+export const AlbumName = ({ singleAlbumItem }) => {
+  //destruct the singleAlbumItem object to extract the 'name' and 'external_urls'
+  const { name, external_urls } = singleAlbumItem;
+  //extract the Spotify URL from the external_urls object.
+  const { spotify } = external_urls;
```

Comment:
Very smart to extract spotify from external_urls! The structure in the return becomes very clear and readable.